lib/schmooze/base.rb

```diff
@@ -96,7 +96,7 @@ def ensure_packages_are_initiated(stdin, stdout, stderr, process_thread)
           process_thread.join
 
           error_message = result[1]
-          if /\AError: Cannot find module '(.*)'\z/ =~ error_message
+          if /\AError: Cannot find module '(.*)'/ =~ error_message
```

Comment:
I wonder why `\z` started failing in the tests and why the solution was to remove it all together vs using `\Z` or `$`?

Reply:
`\z` started failing since it wasn't the end of the string. I removed it since the conditional was just checking if the message was a module not found error and the `\z` wasn't adding any significance. 

`\Z` doesn't work either as there's a `\n` and more content afterwards (`\Z` matches the end of the string with optional new line). `$` does work as it matches up to the newline. I added that in!

Example error:
"Error: Cannot find module 'this-package-is-not-here'\nRequire stack:\n- /Users/jennyshen/src/github.com/Shopify/schmooze/test/fixtures/uninstalled_package/[eval]"